### PR TITLE
created wpf ui for create event story

### PR DESCRIPTION
**Pull request by Jensen96**

### CODE REVIEW

File: Frontend/Frontend/Desktop/CreateEvent.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                tbEventLocation.Text.Length != 0 &&
+                tbEventDescription.Text.Length != 0 &&
+                tbEventMinPrice.Text.Length != 0 &&
+                tbEventMaxPrice.Text.Length != 0 &&
+                tbEventNumOfParticipants.Text.Length != 0
+                )
+            {
+                DateTime datetime = dtpEventDatetime.Value.Value;
+                bool isPublic = false;
+                if (rbEventPublic.IsChecked == true)
```
Comment (Arcanedarkness) [inline Frontend/Frontend/Desktop/CreateEvent.xaml.cs]: Could you not just do `isPublic = rbEventPublic.IsChecked` ?

**Review by Arcanedarkness — APPROVED** [review]

File: Frontend/Frontend/Desktop/MainWindow.xaml.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public MainWindow()
         }
          private void Load_Window(object sender, RoutedEventArgs e)
         {
-           MainFrame.NavigationService.Navigate(new UserLogIn());
```
Comment (refnedergaard) [inline Frontend/Frontend/Desktop/MainWindow.xaml.cs]: This is not right... Startup page must be UserLogIn.

File: Frontend/Frontend/Desktop/CreateEvent.xaml.cs (hunk 1 above)
Comment (Arcanedarkness) [reply Frontend/Frontend/Desktop/CreateEvent.xaml.cs]: test

**Review by Arcanedarkness — CHANGES_REQUESTED** [review]
test

**Review by Arcanedarkness — APPROVED** [review]

**Pull request merged**